### PR TITLE
fix: add sig name for members and only reutrn public sigs

### PR DESCRIPTION
**Pull request by hi-rustin**

### CODE REVIEW

**Comment by hi-rustin** [comment]
@Mini256  Could you please take a look?

File: src/services/sig/index.ts [hunk 1]
```diff
@@ -206,8 +206,18 @@ export class SigService implements ISigService {
   public async listSigs(
```
Comment (Mini256) [inline src/services/sig/index.ts:206]: Maybe the comment of the function should be modified.

**Review by Mini256 — CHANGES_REQUESTED** [review]
rest LGTM

File: src/services/sig/index.ts (hunk 1 above)
Comment (hi-rustin) [reply src/services/sig/index.ts:206]: Good catch! Fixed.

Comment (hi-rustin) [reply src/services/sig/index.ts:206]: Thanks for you review.

**Review by Mini256 — APPROVED** [review]

**Pull request merged**